docs/source/conf.py

```diff
@@ -42,7 +44,7 @@
     'sphinx_rtd_theme',
     'sphinx.ext.napoleon',
     'sphinx.ext.autosummary',
-    'sphinxcontrib.programoutput',
+    "sphinx.ext.githubpages"
```

Comment:
Is this being used? To my knowledge, we are not using github pages?

Reply:
I confused its functionality with something else.
Removed !